includes/class-wc-checkout.php

```diff
@@ -710,33 +710,41 @@ protected function validate_checkout( &$data, &$errors ) {
 	}
 
 	/**
+	 * Set address field for customer.
+	 *
+	 * @since 3.0.7
+	 * @param $field string to update
+	 * @param $key
+	 * @param $data array of data to get the value from
+	 */
+	protected function set_customer_address_fields( $field, $key, $data ) {
```

Comment:
Trivial nitpick, but I think this should be `set_customer_address_field` (singular). It only sets one field. :)

Everything else looks good.